app/Config.hs

```diff
@@ -44,8 +43,13 @@ data CopyThing = CopyThing FilePath FilePath
 instance FromJSON Config where
   parseJSON = withObject "Hapistrano configuration" $ \o -> do
     configDeployPath <- o .: "deploy_path"
-    configHost       <- o .:? "host"
-    configPort       <- o .:? "port" .!= 22
+    host             <- o .:? "host"
+    port             <- o .:? "port" .!= 22
+    hs               <- (o .:? "targets" .!= []) >>= mapM (\m -> do
+      host' <- m .: "host"
+      port' <- m .:? "port" .!= 22
+      return (host', port'))
+    let configHosts = maybeToList ((,) <$> host <*> pure port) ++ hs
```

Comment:
@mrkkrp are we merging hosts for backward compatibility only? What do you think about just keeping `targets` to simplify things and prevent misunderstandings from the user perspective?